docs/faq.rst

```diff
@@ -104,6 +104,19 @@ Anyone interested in using v3.2 may compile and install an earlier release (<=0.
 .. _afni_proc.py: https://afni.nimh.nih.gov/pub/dist/doc/program_help/afni_proc.py.html
 
 
+***********************************************************
+[tedana] How do I perform RICA (manual ICA classification)?
+***********************************************************
```

Comment:
I would recommend making this FAQ more general (e.g., "How do I perform manual ICA classification?" or "How do I change tedana's classifications?"), with some information on manacc and related parameters, and then include the paragraph about RICA (specifically saying that it's our recommended method for manual classification).

Reply:
In #823 I added an anchor to an earlier question in the FAQ. It would be great if you could move that anchor to point to this question. Also, I think you could remove that old question ("[tedana] I think that some BOLD ICA components have been misclassified as noise."), or maybe just use it to link to this one.

Reply:
This whole paragraph will need to change when #756 is merged. There will no longer be a `--manacc` in `tedana` and there will be a separate workflow `tedana_reclassify` to use for changing component classifications after an automated decision tree was run. (FWIW, the reason for this change was that hundreds of lines of code in tedana.py were there solely for this option and it made the code very hard to understand) @eurunuela and I have talked about this a while ago & this will probably require some changes to rica. I just added the following text to the FAQ in #756

********************************************************************************
[tedana] Can I manually reclassify components?
********************************************************************************

``tedana_reclassify`` allows users to manually alter component classifications.
This can both be used as a command line tool or as part of other interactive
programs, such as `RICA`_. RICA creates a graphical interface that is similar to
the build-in tedana reports that lets users interactively change component
classifications. Both programs will log which component classifications were
manually altered. If one wants to retain the original denoised time series,
make sure to output the denoised time series into a separate directory.

.. _RICA: https://github.com/ME-ICA/rica